src/System.Net.Mail/tests/Functional/AlternateViewCollectionTest.cs

```diff
@@ -0,0 +1,65 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
+//
+// AlternateViewCollectionTest.cs - NUnit Test Cases for System.Net.MailAddress.AlternateViewCollection
+//
+// Authors:
+//   John Luke (john.luke@gmail.com)
+//
+// (C) 2005 John Luke
```

Comment:
The guidance I got from @richlander was to keep the entire header for code we take from Mono.